docs/user-guide/api-mediation/routing-requests-to-rest-apis.md

```diff
@@ -0,0 +1,176 @@
+
+Types of services:
+* Versioned services - Routing with serviceId and version
+* Nonversioned services - Using only the service Id
+
+Under certain conditions it's possible to route to a specific instance of service.
+
+## Terminology
+
+* **Service**
+
+  * A service provides one or more APIs, and is identified by a service ID. Note that sometimes the term "service name" is used to mean service ID.  
+  * The default service ID is provided by the service developer in the service configuration file.  
+  * A system administrator can replace the service ID with a deployment environment specific name using additional configuration that is external to the service deployment unit. Most often, this is configured in a JAR or WAR file.  
+  * Services are deployed using one or more service instances, which share the same service ID and implementation.
+
+* **URI (Uniform Resource Identifier)**
+
+  A string of characters used to identify a resource. Each URI must point to a single corresponding resource that does not require any additional information, such as HTTP headers.
+
+## Basic Routing
+
+The basic method of routing is based on the service ID. For the services that has multiple versions of the API the secondary parameter is the version of the API that you want to reach. 
+
+### API ML Routing to the Versioned service
+
+The URI identifies the resource, but does not identify the instance of the service as unique when multiple instances of the same service are provided. For example, when a service is running in high-availability (HA) mode. To get to a specific instance, you would need to access it with specific API ML configuration and header X-Instance-Id.
+
+In addition to the basic Zuul routing, the Zowe API Gateway supports versioning in which you can specify a major version. The Gateway routes a request only to an instance that provides the specified major version of the API.
+
+The `/api/` prefix is used for REST APIs. The prefix `/ui/` applies to web UIs, the prefix `/ws/` applies to [WebSockets](websocket.md) and the prefix `/graphql/` applies to GraphQL API.
+
+The URL expected by the API Gateway has the following format:
+
+`https://{gatewayHost}:{port}/{serviceId}/api/v{majorVersion}/{resource}`
+
+**Example:**
+
+The following address shows the original URL of a resource exposed by a service:
+
+```
+https://service:10015/enablerv1sampleapp/api/v1/samples
+```
+
+The following address shows the API Gateway URL of the resource:
+
+```
+https://gateway:10010/enablerv1sampleapp/api/v1/samples
+```
+
+The following diagram illustrates how basic routing works:
+
+![Zowe API Mediation basic routing](../../images/api-mediation/Basic-Routing.png)
+
+### Implementation details for routing
```

Comment:
I would move this section to the Extend part.

Reply:
A new file entitled "implementing-routing-to-the-api-gateway.md" has been created in this branch and added to the sidebar under the extending section.